package-lock.json

```diff
@@ -0,0 +1,1875 @@
+{
```

Comment:
Looks pretty similar to a `Gemfile.lock` or a `yarn.lock`. We ignore `Gemfile.lock`, should we ignore this file as well?